static/src/javascripts/projects/common/modules/onward/tonal.js

```diff
@@ -6,19 +6,15 @@ import { Component } from 'common/modules/component';
 
 const tones = {
     uk: {
-        features: 'uk-alpha/features/feature-stories',
         comment: 'uk-alpha/contributors/feature-stories',
     },
     us: {
-        features: 'us-alpha/features/feature-stories',
         comment: 'us-alpha/contributors/feature-stories',
     },
     au: {
-        features: 'au-alpha/features/feature-stories',
         comment: 'au-alpha/contributors/feature-stories',
     },
     int: {
-        features: '22167321-f8cf-4f4a-b646-165e5b1e9a30',
```

Comment:
I'm not entirely sure about this one - will have to look.

Reply:
In fact, it doesn't like any of the other editions show a container here suggesting the containers no longer exist.